packages/react-dom/src/__tests__/ReactDOMServerIntegrationAttributes-test.js

```diff
@@ -19,7 +20,7 @@ let ReactDOMServer;
 
 function initModules() {
   // Reset warning cache.
-  jest.resetModuleRegistry();
```

Comment:
"jest.resetModuleRegistry" has been removed in Jest v27 and should be replaced with "jest.resetModules".